src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationService.kt

```diff
@@ -355,6 +360,21 @@ internal class RecommendationService(
     }
   }
 
+  private fun sendRecommendationStartedEvent(recommendationId: Long) {
+    val crn = recommendationRepository.findById(recommendationId).map { it.data.crn }.get()
+    val payload = MrdEvent(
+      message = MrdEventMessageBody(
+        eventType = "RECOMMENDATION_STARTED",
+        version = 1,
+        description = "Recommendation started",
```

Comment:
Think this needs to be "Recommendation started (recall or no recall)" to match the contract in MRD-921

Reply:
Corrected